PrestaShop/PrestaShop
# Fix duplicate payment submissions and address initialization

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fixing bug introduced from commit 70d1104a62da025fca57138790b3eba9a3040895  Follows: 1.7.0.0-beta.4.0, 1.7.0.0-rc.0.0 Precedes: 1.7.0.0-rc.1.0, 1.7.0.0-rc.2.0. Currently when you finish an order the payment form is submitted twice. The bug appears to be a copy-paste error.
| Type?         | bug fix 
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 

How to test?  
From a fresh install of 1.7.0.4, while checking out, Choose the "Pay By Check". Open chrome console, and enter 

> $("#payment-form").submit(function(){ alert("submit") })

When you finish the order by clicking "ORDER WITH AN OBLIGATION TO PAY" you will see that the submit alert appears twice.  With this bug fix, you can verify that the form is only submitted once.
